components/datetime/src/options/components.rs

```diff
@@ -338,6 +340,22 @@ pub enum Text {
     Narrow,
 }
 
+/// Options for displaying a Year for the `components::`[`Bag`].
+#[derive(Debug, Clone, Copy, PartialEq)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
```

Comment:
> suggestion

We should add

```rust
#[serde(rename_all = "camelCase")]
```

This should fix the inconsistent JSON data that contains both `"Numeric"` and `"numeric"`.

Reply:
Ah, I see my mistake. @gregtatum did already add renaming for option components and @mildgravitas was just following that, and I was unaware because the diff only showed the additions of renames.

I think I'd generally prefer not to rename anything unnecessairly, but in this case, I can imagine users wanting to react options bags from JSON, so I'd be okay to use renames.

My only recommendation would be to use `kebab-case` for all of them - @mildgravitas can you update all of component enums (including the one you're adding) to use `kebab-case`?

Reply:
Done: I've added rename_all = kebab-case to the Year & Week enums.

Reply:
And the same for all other component enums.